functions/poll.js

```diff
@@ -529,4 +533,16 @@ module.exports = {
     }
     return null;
   }),
+  // Regenerates the bigquery tables at midnight every day
+  dumpToBigQuery: functions.runWith(
+    // Set to the maximum timeout and memory usage
+    {
+      timeoutSeconds: 540,
+      memory: '2GB'
+    }
+  ).pubsub.schedule('0 0 * * *').timeZone('America/New_York').onRun(async () => {
```

Comment:
maybe an improvement for later (lol) but I wonder if it would be a little more stable to generate a scheduled method for each of the 3 tables and pass the table name in as part of that. that way the timeout can be a little lower AND if one table fails b/c of an airtable schema getting messed up or something, you only need to retry that table, and not do all 3 over

Reply:
Agreed